demisto_sdk/commands/create_content_graph/constants.py

```diff
@@ -0,0 +1,85 @@
+
+import enum
+from pathlib import Path
+
+
+PACKS_FOLDER = 'Packs'
+PACK = 'Pack'
+COMMAND = 'Command'
+
+
+class Marketplaces(enum.Enum):
+    XSOAR = 'xsoar'
+    XSIAM = 'marketplacev2'
```

Comment:
can we use existing constants instead? (same for all constants we have here, most already exist as `Enum`s)